### ISSUE TITLE
Postage stamp - Price calculation seems wrong

### ISSUE THREAD

**Issue by Cafe137** [ethersphere/bee-dashboard#303]

**Comment by agazso** [comment]
When trying to buy a postage stamp with `depth=20` and `amount=1_000_000_000` the indicative price is `0.000006400 BZZ`. I have 0.08951 BZZ and 0.00153 XDAI, so this should not be a problem but Bee reports that I have insufficient token balance. I managed to buy a postage stamp with the same depth with `amount=100_000_000`.

Blockscout link: https://blockscout.com/xdai/mainnet/address/0xaDB5f7402Dc0Aa7c94764edC527F1BEdc54430E3/transactions

Bee log:
```
DEBU[2022-03-03T11:43:02+01:00] create batch: out of funds: insufficient token balance 
ERRO[2022-03-03T11:43:02+01:00] create batch: out of funds                   
```

<img width="1078" alt="Screenshot 2022-03-03 at 11 45 49" src="https://user-images.githubusercontent.com/230163/156549469-9aee1a04-98a8-4fb7-8f9f-08a08b2063ee.png">

**Comment by vojtechsimetka** [comment]
Fixed by #305
